index.html

```diff
@@ -2479,9 +2479,14 @@ <h1>Privacy Considerations</h1>
 	<p class="issue">TODO: Define privacy considerations for DID resolution</p>
 
 	<section id="profiling-requesters">
-		<h2>Non-DID Identifiers</h2>
-		<p class="issue" data-number="95">There is discussion about minimizing the extent of profiling of requesters
-		from DID Resolvers.
+		<h2>Profiling of DID Resolution and Dereferencing Requestors</h2>
+		<p><a>DID resolvers</a> and <a>DID URL dereferencers</a> will be able to log resolution
+			and dereferencing requests to their services. Over time these logs could be used to track 
+			and profile clients making requests to these services. To mitigate this privacy risk, 
+			clients should make requests to services they trust, for example 
+			because of an existing business relationship or because the service is running on 
+			infrastructure they control. Additionally, clients may take steps to obfuscate their 
+			requests to a service in order to limit the possibilities for correlation and profiling.
```

Comment:
```suggestion
		<h2>Profiling of DID Resolution and Dereferencing Requesters</h2>
		<p><a>DID resolvers</a> and <a>DID URL dereferencers</a> will be able to log requests
			to their services for resolution and dereferencing. Over time, these logs could be used to track 
			and profile the clients making requests for these services. To mitigate this privacy risk, 
			clients should make such requests to services they trust, for example, 
			because of an existing business relationship or because the service is running on 
			infrastructure they control. Clients can also take steps to obfuscate their 
			requests to a service in order to limit the possibilities of correlation and profiling.
```